core/backend/src/IModelJsFs.ts

```diff
@@ -143,7 +143,14 @@ export class IModelJsFs {
     const parentPath = path.dirname(dirPath);
     if (parentPath !== dirPath)
       IModelJsFs.recursiveMkDirSync(parentPath);
-    IModelJsFs.mkdirSync(dirPath);
+    try {
+      IModelJsFs.mkdirSync(dirPath);
+    } catch (err: any) {
+      // Ignore the error if the folder has been created since the existence check
+      if (err?.code !== 'EEXIST') {
+      throw err;
```

Comment:
```suggestion
        throw err;
```

Reply:
Yes. I didn't have the right node version to run it. Working on it...

Reply:
All resolved. Just an FYI that the linter didn't catch the missing indent or that I used single-quotes. Not sure if that's a problem...